### ISSUE TITLE
[Bug] ClusterCleanupPolicy works only the first time ?

### ISSUE THREAD

**Issue by amoscatellialmaviva** [kyverno/policies#1053]
### Kyverno Version

1.12

### Kubernetes Version

1.28

### Kubernetes Platform

VMware Tanzu (specify in description)

### Description

For some strange reason my ClusterCleanupPolicy seems to be applied only once ...

Here are my yamls :

```
apiVersion: kyverno.io/v2beta1
kind: ClusterPolicy
metadata:
  name: mutate-pod-binding
spec:
  background: false
  rules:
    - name: mutate-pod-binding
      match:
        any:
        - resources:
            kinds:
            - Pod/binding
            namespaces:
            - shm
            - device-manager
            - p3p
      context:
      - name: node
        variable:
          jmesPath: request.object.target.name
          default: ''
      - name: assetCode
        apiCall:
          urlPath: "/api/v1/nodes/{{node}}"
          jmesPath: "metadata.labels.\"asset-code\" || 'empty'"
      - name: assetName
        apiCall:
          urlPath: "/api/v1/nodes/{{node}}"
          jmesPath: "metadata.labels.\"asset-name\" || 'empty'"
      mutate:
        patchStrategicMerge:
          metadata:
            annotations:
              asset-code: "{{ assetCode }}"
              asset-name: "{{ assetName }}"
              asset: "initialized"
---
apiVersion: kyverno.io/v2beta1
kind: ClusterCleanupPolicy
metadata:
  name: cleandeploy
spec:
  match:
    any:
    - resources:
        kinds:
          - Pod
        namespaces:
        - shm
        - device-manager
        - p3p
  exclude:
    any:
    - resources:
        annotations:
          asset: "initialized"
  schedule: "*/1 * * * *"
```

I can see it has been started the first time : 

```
Normal  PolicyApplied  12m   kyverno-cleanup  successfully cleaned up the target resource Pod/shm/dfm-5b5578b5d4-wl9tr
```

Yet now I see this :

```
NAME                   READY   STATUS             RESTARTS        AGE
dfm-5b5578b5d4-5247b   0/1     CrashLoopBackOff   7 (97s ago)     14m
```

```
Name:             dfm-5b5578b5d4-5247b
Namespace:        shm
Priority:         0
Service Account:  default
Node:             ec-worker/10.238.3.249
Start Time:       Wed, 19 Jun 2024 11:50:39 +0000
Labels:           app=am
                  pod-template-hash=6c9c85bdd8
Annotations:      data-ingest.dynatrace.com/injected: true
                  dynakube.dynatrace.com/injected: true
                  oneagent.dynatrace.com/injected: true
```

But I can't understand why the ClusterCleanupPolicy is not triggered anymore, and the describe proove it since there are not new events.

If I REDEPLOY the policy it's started an additional times ...

Why ?

### Steps to reproduce

I provided everything in the description

### Expected behavior

I provided everything in the description

### Troubleshooting

- [X] I have read and followed the documentation AND the [troubleshooting guide](https://kyverno.io/docs/troubleshooting/).
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by amoscatellialmaviva** [comment]
> Duplicate of [kyverno/kyverno#10418](https://github.com/kyverno/kyverno/issues/10418)

It's true that I can see those "not found errors" in the log, yet those lines aren't related to the the 'dfm-' pod.

Are you sure this is a duplicate ?

```
Defaulted container "controller" out of: controller, install-oneagent (init)
2024-06-19T11:50:21Z    INFO    setup.version   version/version.go:49   version {"version": "v1.12.0", "hash": "111b052c9a1826a563d1763b6d302e31a5338aa0", "time": "2024-04-22T07:28:05Z"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"add_dir_header": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"alsologtostderr": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"caSecretName": "kyverno-cleanup-controller.kyverno.svc.kyverno-tls-ca"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"cleanupServerPort": "9543"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"clientRateLimitBurst": "50"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"clientRateLimitQPS": "20"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"disableMetrics": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"dumpPayload": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"enableConfigMapCaching": "true"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"enableDeferredLoading": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"enableTracing": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"eventsRateLimitBurst": "2000"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"eventsRateLimitQPS": "1000"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"kubeconfig": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"leaderElectionRetryPeriod": "2s"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"log_backtrace_at": ":0"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"log_dir": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"log_file": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"log_file_max_size": "1800"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"loggingFormat": "text"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"loggingtsFormat": "default"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"logtostderr": "true"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"maxAPICallResponseLength": "2000000"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"maxQueuedEvents": "1000"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"metricsPort": "8100"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"one_output": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"otelCollector": "opentelemetrycollector.kyverno.svc.cluster.local"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"otelConfig": "prometheus"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"profile": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"profileAddress": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"profilePort": "6060"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"protectManagedResources": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"renewBefore": "360h0m0s"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"serverIP": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"servicePort": "443"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"skip_headers": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"skip_log_headers": "false"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"stderrthreshold": "2"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"tlsSecretName": "kyverno-cleanup-controller.kyverno.svc.kyverno-tls-pair"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"tracingAddress": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"tracingCreds": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"tracingPort": "4317"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"transportCreds": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"ttlReconciliationInterval": "1m0s"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"v": "2"}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"vmodule": ""}
2024-06-19T11:50:21Z    LEVEL(-2)       setup.flag      internal/flag.go:260            {"webhookServerPort": "9543"}
2024-06-19T11:50:21Z    INFO    setup.maxprocs  internal/maxprocs.go:12 setup maxprocs...
2024-06-19T11:50:21Z    INFO    setup.maxprocs  internal/maxprocs.go:16 maxprocs: Leaving GOMAXPROCS=4: CPU quota undefined
2024-06-19T11:50:21Z    INFO    setup.signals   internal/signal.go:16   setup signals...
2024-06-19T11:50:21Z    INFO    setup.kube-client       internal/client.go:43   create kube client...   {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.ConfigMap from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:21Z    INFO    config  config/metricsconfig.go:153     metricsRefreshInterval not set  {"name": "kyverno-metrics", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    config  config/metricsconfig.go:175     namespaces configured   {"name": "kyverno-metrics", "namespace": "kyverno", "namespaces": "{\"exclude\":[],\"include\":[]}"}
2024-06-19T11:50:21Z    INFO    config  config/metricsconfig.go:189     bucketBoundaries configured     {"name": "kyverno-metrics", "namespace": "kyverno", "bucketBoundaries": "0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 15, 20, 25, 30"}
2024-06-19T11:50:21Z    INFO    config  config/metricsconfig.go:195     metricsExposure not set {"name": "kyverno-metrics", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    setup.metrics   internal/metrics.go:18  setup metrics...        {"otel": "prometheus", "port": "8100", "collector": "opentelemetrycollector.kyverno.svc.cluster.local", "creds": ""}
2024-06-19T11:50:21Z    DEBUG   metrics-config-controller       controller/run.go:58    starting ...
2024-06-19T11:50:21Z    DEBUG   metrics-config-controller.worker        controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:21Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.ConfigMap from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:21Z    INFO    config  config/config.go:355    filters configured      {"name": "kyverno", "namespace": "kyverno", "filters": [{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kyverno","Name":"*"},{"Group":"*","Version":"*","Kind":"Event","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-system","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-public","Name":"*"},{"Group":"*","Version":"*","Kind":"*","Subresource":"*","Namespace":"kube-node-lease","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"Node","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"APIService","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"TokenReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"SelfSubjectAccessReview","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ReplicaSet","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"AdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterAdmissionReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"BackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterBackgroundScanReport","Subresource":"*","Namespace":"*","Name":"*"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:background-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:core"},{"Group":"*","Version":"*","Kind":"ClusterRole","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller:additional"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"ClusterRoleBinding","Subresource":"","Namespace":"*","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"ServiceAccount","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"Role","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:admission-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:background-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:cleanup-controller"},{"Group":"*","Version":"*","Kind":"RoleBinding","Subresource":"","Namespace":"kyverno","Name":"kyverno:reports-controller"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno"},{"Group":"*","Version":"*","Kind":"ConfigMap","Subresource":"","Namespace":"kyverno","Name":"kyverno-metrics"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Deployment","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Pod","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-*"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"Job","Subresource":"*","Namespace":"kyverno","Name":"kyverno-hook-pre-delete"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"NetworkPolicy","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"PodDisruptionBudget","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-svc-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-background-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-cleanup-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"Service","Subresource":"*","Namespace":"kyverno","Name":"kyverno-reports-controller-metrics"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-admission-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-background-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller"},{"Group":"*","Version":"*","Kind":"ServiceMonitor","Subresource":"","Namespace":"kyverno","Name":"kyverno-reports-controller"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-svc.kyverno.svc.*"},{"Group":"*","Version":"*","Kind":"Secret","Subresource":"","Namespace":"kyverno","Name":"kyverno-cleanup-controller.kyverno.svc.*"}]}
2024-06-19T11:50:21Z    INFO    config  config/config.go:364    defaultRegistry configured      {"name": "kyverno", "namespace": "kyverno", "defaultRegistry": "docker.io"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:380    enableDefaultRegistryMutation configured        {"name": "kyverno", "namespace": "kyverno", "enableDefaultRegistryMutation": "true"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:389    excludedGroups configured       {"name": "kyverno", "namespace": "kyverno", "excludeGroups": ["system:nodes"], "includeGroups": []}
2024-06-19T11:50:21Z    INFO    config  config/config.go:394    excludeUsernames not set        {"name": "kyverno", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:402    excludeRoles not set    {"name": "kyverno", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:410    excludeClusterRoles not set     {"name": "kyverno", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:426    generateSuccessEvents configured        {"name": "kyverno", "namespace": "kyverno", "generateSuccessEvents": "false"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:440    webhooks configured     {"name": "kyverno", "namespace": "kyverno", "webhooks": "[{\"namespaceSelector\":{\"matchExpressions\":[{\"key\":\"kubernetes.io/metadata.name\",\"operator\":\"NotIn\",\"values\":[\"kube-system\"]},{\"key\":\"kubernetes.io/metadata.name\",\"operator\":\"NotIn\",\"values\":[\"kyverno\"]}],\"matchLabels\":null}}]"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:454    webhookAnnotations configured   {"name": "kyverno", "namespace": "kyverno", "webhookAnnotations": "{\"admissions.enforcer/disabled\":\"true\"}"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:460    webhookLabels not set   {"name": "kyverno", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    config  config/config.go:474    matchConditions not set {"name": "kyverno", "namespace": "kyverno"}
2024-06-19T11:50:21Z    INFO    setup.kube-client       internal/client.go:43   create kube client...   {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    DEBUG   config-controller       controller/run.go:58    starting ...
2024-06-19T11:50:21Z    DEBUG   config-controller.worker        controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:21Z    INFO    setup.kyverno-client    internal/client.go:51   create kyverno client...        {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.dynamic-client    internal/client.go:59   create dynamic client...        {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.apiserver-client  internal/client.go:75   create apiserver client...      {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.d-client  internal/client.go:83   create the kyverno dynamic client...    {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.events-client     internal/client.go:91   create the events client...     {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.events-client.kube-client internal/client.go:43   create kube client...   {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:21Z    INFO    setup.metadata-client   internal/client.go:67   create metadata client...       {"kubeconfig": "", "qps": 20, "burst": 50}
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.Secret from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.Secret from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v2beta1.ClusterCleanupPolicy from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    INFO    setup.cluster-cleanup-policy    logging/controller.go:45        resource added  {"type": "ClusterCleanupPolicy", "name": "cleandeploy"}
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v2beta1.CleanupPolicy from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v2alpha1.GlobalContextEntry from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.Namespace from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:22Z    INFO    klog    leaderelection/leaderelection.go:250    attempting to acquire leader lease kyverno/kyverno-cleanup-controller...
2024-06-19T11:50:22Z    INFO    setup   internal/controller.go:32       starting controller     {"name": "kyverno-events", "workers": 3}
2024-06-19T11:50:22Z    INFO    EventGenerator  event/controller.go:101 start
2024-06-19T11:50:22Z    INFO    setup   internal/controller.go:32       starting controller     {"name": "global-context", "workers": 1}
2024-06-19T11:50:22Z    DEBUG   global-context  controller/run.go:58    starting ...
2024-06-19T11:50:22Z    DEBUG   global-context.worker   controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:22Z    INFO    setup.leader-election   leaderelection/leaderelection.go:99     another instance has been elected as leader     {"id": "kyverno-cleanup-controller-b55d69b6b-fkgct", "leader": "kyverno-cleanup-controller-b55d69b6b-9jsgz"}
2024-06-19T11:50:36Z    INFO    klog    leaderelection/leaderelection.go:260    successfully acquired lease kyverno/kyverno-cleanup-controller
2024-06-19T11:50:36Z    INFO    setup.leader-election   leaderelection/leaderelection.go:97     still leading   {"id": "kyverno-cleanup-controller-b55d69b6b-fkgct"}
2024-06-19T11:50:36Z    INFO    setup.leader-election   leaderelection/leaderelection.go:83     started leading {"id": "kyverno-cleanup-controller-b55d69b6b-fkgct"}
2024-06-19T11:50:36Z    INFO    setup.configmap-resolver        internal/engine.go:90   setup config map resolver...    {"enableConfigMapCaching": true}
2024-06-19T11:50:36Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.ConfigMap from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:36Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v2beta1.ClusterCleanupPolicy from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:36Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v2beta1.CleanupPolicy from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:36Z    INFO    cleanup-controller      cleanup/controller.go:97        added   {"kind": "ClusterCleanupPolicy", "operation": "added", "name": "cleandeploy"}
2024-06-19T11:50:36Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.ValidatingWebhookConfiguration from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:50:36Z    INFO    setup.leader    internal/controller.go:32       starting controller     {"name": "ttl-controller-manager", "workers": 3}
2024-06-19T11:50:36Z    INFO    setup.leader    internal/controller.go:32       starting controller     {"name": "certmanager-controller", "workers": 1}
2024-06-19T11:50:36Z    DEBUG   certmanager-controller  controller/run.go:58    starting ...
2024-06-19T11:50:36Z    DEBUG   certmanager-controller.routine  controller/run.go:80    starting routine        {"id": 0}
2024-06-19T11:50:36Z    INFO    setup.leader    internal/controller.go:32       starting controller     {"name": "policy-webhook-controller", "workers": 2}
2024-06-19T11:50:36Z    DEBUG   policy-webhook-controller       controller/run.go:58    starting ...
2024-06-19T11:50:36Z    DEBUG   policy-webhook-controller.worker        controller/run.go:71    starting worker {"id": 1}
2024-06-19T11:50:36Z    INFO    setup.leader    internal/controller.go:32       starting controller     {"name": "cleanup-controller", "workers": 3}
2024-06-19T11:50:36Z    DEBUG   certmanager-controller.worker   controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:36Z    DEBUG   policy-webhook-controller.worker        controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:36Z    INFO    setup.leader    internal/controller.go:32       starting controller     {"name": "ttl-webhook-controller", "workers": 2}
2024-06-19T11:50:36Z    DEBUG   ttl-webhook-controller  controller/run.go:58    starting ...
2024-06-19T11:50:36Z    DEBUG   ttl-webhook-controller.worker   controller/run.go:71    starting worker {"id": 1}
2024-06-19T11:50:36Z    DEBUG   ttl-webhook-controller.worker   controller/run.go:71    starting worker {"id": 0}
2024-06-19T11:50:38Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy"}
2024-06-19T11:50:39Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T11:50:40Z    ERROR   cleanup-controller.worker       cleanup/controller.go:407       failed to update the cleanup policy status      {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "error": "Operation cannot be fulfilled on clustercleanuppolicies.kyverno.io \"cleandeploy\": the object has been modified; please apply your changes to the latest version and try again"}
2024-06-19T11:50:41Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T11:50:42Z    ERROR   cleanup-controller.worker       cleanup/controller.go:338       failed to delete resource       {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "core-metadata-7fbc89c6bb-rgp2f", "namespace": "device-manager", "error": "pods \"core-metadata-7fbc89c6bb-rgp2f\" not found"}
2024-06-19T11:50:42Z    ERROR   cleanup-controller.worker       cleanup/controller.go:338       failed to delete resource       {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "rule-engine-proxy-d8699c4b7-m74sn", "namespace": "device-manager", "error": "pods \"rule-engine-proxy-d8699c4b7-m74sn\" not found"}
2024-06-19T11:50:42Z    ERROR   cleanup-controller.worker       cleanup/controller.go:338       failed to delete resource       {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "p3p-hub-5797b565bd-zv4l4", "namespace": "p3p", "error": "pods \"p3p-hub-5797b565bd-zv4l4\" not found"}
2024-06-19T11:50:42Z    ERROR   cleanup-controller.worker       cleanup/controller.go:338       failed to delete resource       {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "am-6c9c85bdd8-4xmcg", "namespace": "shm", "error": "pods \"am-6c9c85bdd8-4xmcg\" not found"}
2024-06-19T11:50:43Z    ERROR   cleanup-controller.worker       cleanup/controller.go:338       failed to delete resource       {"id": 2, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "dm-7c4dff54fd-hn54d", "namespace": "shm", "error": "pods \"dm-7c4dff54fd-hn54d\" not found"}
2024-06-19T11:51:39Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.PartialObjectMetadata from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:51:39Z    INFO    ttl-controller-manager  controller/run.go:58    starting ...    {"gvr": "/v1, Resource=pods"}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "/v1, Resource=pods", "id": 2}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "/v1, Resource=pods", "id": 0}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "/v1, Resource=pods", "id": 1}
2024-06-19T11:51:39Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.PartialObjectMetadata from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:51:39Z    INFO    ttl-controller-manager  controller/run.go:58    starting ...    {"gvr": "admissionregistration.k8s.io/v1, Resource=validatingwebhookconfigurations"}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "admissionregistration.k8s.io/v1, Resource=validatingwebhookconfigurations", "id": 2}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "admissionregistration.k8s.io/v1, Resource=validatingwebhookconfigurations", "id": 0}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "admissionregistration.k8s.io/v1, Resource=validatingwebhookconfigurations", "id": 1}
2024-06-19T11:51:39Z    LEVEL(-2)       klog    cache/reflector.go:351  Caches populated for *v1.PartialObjectMetadata from k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229
2024-06-19T11:51:39Z    INFO    ttl-controller-manager  controller/run.go:58    starting ...    {"gvr": "kyverno.io/v2alpha1, Resource=globalcontextentries"}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "kyverno.io/v2alpha1, Resource=globalcontextentries", "id": 2}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "kyverno.io/v2alpha1, Resource=globalcontextentries", "id": 0}
2024-06-19T11:51:39Z    INFO    ttl-controller-manager.worker   controller/run.go:71    starting worker {"gvr": "kyverno.io/v2alpha1, Resource=globalcontextentries", "id": 1}
2024-06-19T12:06:38Z    LEVEL(-2)       klog    rest/request.go:697     Waited for 1.044723703s due to client-side throttling, not priority and fairness, request: GET:https://10.96.0.1:443/apis/source.toolkit.fluxcd.io/v1beta1
2024-06-19T12:06:39Z    LEVEL(-2)       klog    rest/request.go:697     Waited for 2.044812785s due to client-side throttling, not priority and fairness, request: GET:https://10.96.0.1:443/apis/autoscaling/v1
2024-06-19T12:12:55Z    INFO    setup.cluster-cleanup-policy    logging/controller.go:59        resource updated        {"type": "ClusterCleanupPolicy", "name": "cleandeploy"}
2024-06-19T12:12:55Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:12:57Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:12:58Z    ERROR   cleanup-controller.worker       cleanup/controller.go:407       failed to update the cleanup policy status      {"id": 1, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "error": "Operation cannot be fulfilled on clustercleanuppolicies.kyverno.io \"cleandeploy\": the object has been modified; please apply your changes to the latest version and try again"}
2024-06-19T12:12:58Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:12:59Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:12:59Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:00Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:00Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:01Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:02Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:03Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:04Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:05Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:06Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:06Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:07Z    INFO    cleanup-controller      cleanup/controller.go:97        updated {"kind": "ClusterCleanupPolicy", "operation": "updated", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy", "name": "cleandeploy"}
2024-06-19T12:13:08Z    ERROR   cleanup-controller.worker       cleanup/controller.go:366       failed to delete resource       {"id": 1, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "rule-engine-58977dd65c-lz64h", "namespace": "device-manager", "error": "pods \"rule-engine-58977dd65c-lz64h\" not found"}
2024-06-19T12:13:08Z    ERROR   cleanup-controller.worker       cleanup/controller.go:366       failed to delete resource       {"id": 1, "key": "cleandeploy", "namespace": "", "name": "cleandeploy", "kind": "Pod", "name": "support-fileserver-6675fc6b7f-jwdqh", "namespace": "device-manager", "error": "pods \"support-fileserver-6675fc6b7f-jwdqh\" not found"}
```
